wine-key/public/app.js

```diff
@@ -0,0 +1,102 @@
+Parse.initialize("JRtMGAPLmTepO6hBHI49utGj5hD0ANu19rtLXPiz", "VVP7pXG6raUCStZB1OqDzGEn1EYpulOQqoAlQL89");
```

Comment:
Checkout `app.run` instead of doing a global. https://docs.angularjs.org/guide/module#module-loading-dependencies